skimage/measure/_regionprops.py

```diff
@@ -358,22 +358,73 @@ def __eq__(self, other):
         return True
 
 
+def to_dict(regions, want=['label', 'bbox'], separator='-', cancel_auto=False):
+    """Convert image regions properties into a dictionary
+
+    Parameters
+    ----------
+    regions : (N,) list
+        _RegionProperties
+    want : list, optional
+        str of a property the user wants to include
+    separator : str, optional
+        used in joining coordinates
```

Comment:
More detail: "Each element of non-scalar properties will be put into its own column, with the index of that element separated from the name by this separator."